nginx_proxy/modsecurity/Dockerfile

```diff
@@ -0,0 +1,93 @@
+ARG BUILD_FROM
```

Comment:
Not sure why that is needed as folder: `nginx_proxy/modsecurity`

Reply:
We moved everything to the main `nginx_proxy/Dockerfile` so the `nginx_proxy/modsecurity` folder is obsolete. It's deleted now.